src/log_file.rs

```diff
@@ -116,10 +59,7 @@ impl LogFd {
     }
 
     pub fn sync(&self) -> IoResult<()> {
-        let start = Instant::now();
-        let res = fsync(self.0).map_err(|e| from_nix_error(e, "fsync"));
-        LOG_SYNC_TIME_HISTOGRAM.observe(start.saturating_elapsed().as_secs_f64());
```

Comment:
keep the metrics?

Reply:
They are actually moved out to file_pipe_log.rs.